README.md

```diff
@@ -24,6 +24,12 @@ be opened using the "Preferences - Browse Packages" option
 in sublime-text's command pallete or through the
 "Preferences" > "Browse Packages".
 
+#### For Sublime Text 2:
+`git clone https://github.com/coala/coala-sublime.git ~/Library/Application\ Support/Sublime\ Text\ 2/Packages/coala`
+
+#### For Sublime Text 3:
+`git clone https://github.com/coala/coala-sublime.git ~/Library/Application\ Support/Sublime\ Text\ 3/Packages/coala`
```

Comment:
Using this command for sublime 3 on antergos (arch linux) didn't work. Sublime doesn't show any coala package